### PR TITLE
Fix wording and whitespacing ##doc

### PR DESCRIPTION
**Pull request by ratijas**
**Checklist**

- [ ] Closing issues: #issue
- [x] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [x] I wrote some lines in the [radare2book](https://github.com/radareorg/radare2book)

**Description**

Hi there. I'm new to radare2, and I'm learning it by radare2book. As I'm going deeper, I started noticing some things require further polishing — both in the book and in radare2 itself. So, here we are.

I'll be updating this PR with little typo fixes and other insignificant stuff. Don't merge it just yet to avoid flood of noisy micro-commits.

### CODE REVIEW

File: doc/fortunes.tips [hunk 1]
```diff
@@ -79,7 +79,7 @@ This code was intentionally left blank, try 'e asm.arch = ws'
 Thanks for using radare2!
 give | and > a try piping and redirection
 Run .dmm* to load the flags of the symbols of all modules loaded in the debugger
-Use V! to enter into the visual panels mode (dwm style)
+Use V! to enter visual panels mode
```
Comment (trufae) [inline doc/fortunes.tips:82]: this has "changed" to something simpler, which i think is good to document:

* v (lowercase) starts panels
* V (uppercase) start visual mode

The V! is still a thing, but imho its easier to just tell people to use v or V and ! to toggle between panels/visual

File: libr/core/cmd.c [hunk 2]
```diff
@@ -2453,6 +2456,25 @@ static int autocomplete_type(const char* strflag) {
 	return R_CORE_AUTOCMPLT_END;
 }
 
+static void cmd_autocomplete_help(RCore *core) {
+	r_core_cmd_help (core, help_msg_triple_exclamation);
+	// non-zero-cost survival without iterators 101
+	const char **help = malloc(R_CORE_AUTOCMPLT_END * sizeof(char *) * 3 + 1);
+	int i, n;
```
Comment (trufae) [inline libr/core/cmd.c:2463]: use size_t instead of int

File: libr/core/cmd.c [hunk 3]
```diff
@@ -2453,6 +2456,25 @@ static int autocomplete_type(const char* strflag) {
 	return R_CORE_AUTOCMPLT_END;
 }
 
+static void cmd_autocomplete_help(RCore *core) {
+	r_core_cmd_help (core, help_msg_triple_exclamation);
+	// non-zero-cost survival without iterators 101
+	const char **help = malloc(R_CORE_AUTOCMPLT_END * sizeof(char *) * 3 + 1);
```
Comment (trufae) [inline libr/core/cmd.c:2462]: missing space before (

**Review by trufae — CHANGES_REQUESTED** [review]

File: doc/fortunes.tips (hunk 1 above)
Comment (ratijas) [reply doc/fortunes.tips:82]: Agreed.

File: libr/core/cmd.c (hunk 3 above)
Comment (ratijas) [reply libr/core/cmd.c:2462]: ah, OK, I'm still trying to get used to this code style.

**Comment by trufae** [comment]
I will be doing the release in few hours. I plan to merge this pr with those typos fixed. You can create a new one after the release and continue improving the help messages.

thank you!

**Comment by ratijas** [comment]
I'd also like to know, how comes that `R_API void r_cons_cmd_help(const char *help[], bool use_color)` and `R_API void r_core_visual_append_help(RStrBuf *p, const char *title, const char **help)` are two entirely different APIs for the very same purpose? Not to mention they handle little details inconsistently.

**Comment by ratijas** [comment]
> I will be doing the release in few hours. I plan to merge this pr with those typos fixed. You can create a new one after the release and continue improving the help messages.
> 
> thank you!

Oh, that's great! Wait a sec, I figured out all these visual modes, I'm fixing the help text right now.

**Comment by ratijas** [comment]
I need another 30 minutes. I figured out how to fix help text rendering & interaction in visual graph &amp; func/var analysis modes.

**Comment by trufae** [comment]
Ping me when ready :) im fixing tons of issues found by coverity

**Comment by ratijas** [comment]
@trufae Ready for review. Still working on a proper commit message though. I'll squash and split it into 2-3 meaningful commits, and force-push in few minutes.

Things to check out: help text and behavior of most visual modes: V (visual), v (panels), v + w (window), v + Enter (zoom), Vv (analysis), Vv + x (xrefs), global help text for v/V.

**Comment by ratijas** [comment]
My memory management is on par with my time management. Those `RStrBuf` buffers obviously were supposed to be freed.

**Review by trufae — APPROVED** [review]
hats off

**Comment by ratijas** [comment]
@trufae If it's not too late, I'd like you to re-merge it (b745bafc1b4216256afa3e8c9c77fb7fef794b35) again. I haven't split it, but at least I fixed my own memory leaks, and finalized commit message.

**Comment by trufae** [comment]
Remerge? Can you open a new pr for it? I will check it out in a couple of hours. Thanks for the heads up!

**Comment by ratijas** [comment]
Uhm... more like a hard rebase or something. But seems like I'm already going to the local wall of shame with all those dirty stuff in the mainline tree. I should've know better than to upload it to a public accessible repo. `s/WIP/R.I.P./gg` haha :)

Here, I rebased it on the latest commit before mine got merged, namely onto d3157e2ed3c437fae3ca6eb2fb8ece682bd0a342. I doubt GitHub would allow such workflow without manual intervention though.

* Only my rebased commit: https://github.com/radareorg/radare2/pull/18081
* My rebased commit plus the rest of the cherry-picked main branch (can be used as a drop-in replacement for `main`, i.e. `git reset --hard`): https://github.com/radareorg/radare2/pull/18082

**Pull request merged**